src/main/java/com/harrison/controllers/EventController.java

```diff
@@ -34,6 +37,14 @@
 	
 	@RequestMapping(value = "eventcomponents", method = RequestMethod.GET)
 	public List<EventComponent> getEvents() {
+	    String[] data = new String[6];
+	    Arrays.sort(data, new Comparator<String>() {
+            @Override
+            public int compare(String o1, String o2) {
+                // TODO Auto-generated method stub
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://ec2-54-187-222-128.us-west-2.compute.amazonaws.com:9000/sonar/coding_rules#rule_key=squid%3AS1135)